terraform/dev/scripts/user-data.sh

```
@@ -0,0 +1,15 @@
+#!/bin/bash
+echo ECS_CLUSTER=${ecs_cluster_name} >> /etc/ecs/ecs.config
```

Shellcheck (0.10.0)

[warning] ecs_cluster_name is referenced but not assigned.

(SC2154)